keps/sig-auth/3331-structured-authentication-configuration/README.md

```diff
@@ -739,49 +674,42 @@ providers such as Okta, Azure AD, etc:
 
 #### Beta
 
-- Gather feedback
 - Complete benchmarks
+  -  Benchmarks are required to see how different CEL expressions affects authentication time.
+     -  There will be a upper bound of 5s for the CEL expression evaluation. We will use the `apiserver_authentication_latency_seconds` metric to monitor this.
 - Add metrics
-- Initial e2e test with an external provider completed and enabled
+- Support > 1 JWT authenticator and > 1 audiences
+- Enable automatic reload of the configuration
```

Comment:
That's great ! No need to nuke the api server on configuration change is nice.

One thing though, what happens if the new configuration is invalid ? Is the previous one kept or does the API server error out ?

Reply:
The last valid config will continue to be used.  Note that a typo in an issuer URL will not be detected since an issuer is explicitly allowed to be offline when an API server is starting up to allow for self hosted IDPs.  We may come up with an approach to make this more robust in the future.  @aramase can you record that as a pre-GA follow-up?

Reply:
added a pre-GA follow-up.